packages/typespec-ts/test/integration/generated/parameters/collection-format/src/index.d.ts

```diff
@@ -105,14 +141,20 @@ export declare interface QuerySsv204Response extends HttpResponse {
     status: "204";
 }
 
+export declare interface QuerySsvColorsQueryParam {
+    value: string[];
+    explode: false;
+    style: "spaceDelimited";
+}
+
 export declare type QuerySsvParameters = QuerySsvQueryParam & RequestParameters;
 
 export declare interface QuerySsvQueryParam {
     queryParameters: QuerySsvQueryParamProperties;
 }
 
 export declare interface QuerySsvQueryParamProperties {
-    colors: string;
+    colors: QuerySsvColorsQueryParam;
```

Comment:
same here.

Reply:
changed the option to compatibilityQueryMultiFormat.